src/Microsoft.ComponentDetection/Program.cs

```diff
@@ -1,13 +1,14 @@
-﻿using System;
-using System.Diagnostics;
-using System.Linq;
-using Microsoft.ComponentDetection.Contracts;
-
-namespace Microsoft.ComponentDetection.Loader
+﻿namespace Microsoft.ComponentDetection
 {
+    using System;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.ComponentDetection.Contracts;
+
     public class Program
     {
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
```

Comment:
Are you aware of how the SBOM tool and other partners consume the package? Do we need this to be a major version breaking change?

Reply:
Yes, all other consumers use the Orchestrator. This entrypoint is for the standalone `component-detection` CLI. Modification of the Orchestrator entrypoint will require a major version bump:

https://github.com/microsoft/component-detection/blob/7c5d7fed6ef1c372331adf612b6bbb674e6aef9d/src/Microsoft.ComponentDetection.Orchestrator/Orchestrator.cs#L30